nixos/modules/installer/cd-dvd/installation-cd-base.nix

```diff
@@ -46,5 +46,12 @@ with lib;
     done
   '';
 
+  environment.defaultPackages = with pkgs; [
+    # Include some version control tools.
```

Comment:
very much a nitpick but remove that comment, otherwise lgtm

Reply:
Will do that later.

Reply:
Done